ios/Podfile.lock

```diff
@@ -566,4 +566,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: c34e2287a9ccaa606aeceab922830efb9a6ff69a
 
-COCOAPODS: 1.9.3
+COCOAPODS: 1.9.1
```

Comment:
Upgrade your cocoapods so you have the `1.9.3` version by running `sudo gem cocoapods`.